include/gfxbase.h

```diff
@@ -212,9 +212,12 @@ class GFXBase : public Adafruit_GFX
         return to16bit(CRGB(code));
     }
 
-    virtual void Clear()
+    virtual void Clear(CRGB color = CRGB::Black)
     {
-        memset(leds, 0, sizeof(CRGB) * _width * _height);
+        if (color == CRGB::Black)
```

Comment:
I think Dave chooses to use `memset()` to set the array that is `leds` to all zeroes when the color to clear to is black, because it's faster than the `fill_solid()` approach.

I think the `leds()` and `fill_solid()` you're talking about is FastLED stuff? Here both are members of our `GFXBase`.

Reply:
Filling with zero is a common case and much faster with memset, so figured it made sense.  A smarter approach might be to check if r == g == b and then fill with that, but it didn't seem common enough to me...